apiary.apib

```diff
@@ -3080,6 +3082,22 @@ Get details for a single project.
                         + Members
                             + decision_maker
                             + member
+            + tracked_time (object)
+                + total (object)
+                    + unit (enum)
+                        + Members
+                            + h - Hours
+                    + value: `10.25` (number)
+                + billable (object)
+                    + unit (enum)
+                        + Members
+                            + h - Hours
+                    + value: `7.50` (number)
```

Comment:
Should this be renamed to `quantity` to keep it consistent with how we'll do it with `projectItems`? We also have a unit and quantity key there

Reply:
No, this is consistent with other endpoints. If anything, we should change it in `projectItems` 🙂